java/client/src/main/java/glide/api/models/commands/scan/HScanOptionsBinary.java

```diff
@@ -11,4 +18,45 @@
  * @see <a href="https://valkey.io/commands/hscan/">valkey.io</a>
  */
 @SuperBuilder
-public class HScanOptionsBinary extends BaseScanOptionsBinary {}
+public class HScanOptionsBinary extends BaseScanOptionsBinary {
```

Comment:
This class add nothing new comparing to `HScanOptions`. You can delete it and use `HScanOptions`.

Reply:
I am leaving it for consistency, and it might change in the future